index.html

```diff
@@ -84,7 +84,7 @@ <h2 class="card-title">Tonic</h2>
         <section class="work-item">
```

Comment:
- [ ] Add a section heading for this project section. This will help with user navigation